.github/pull_request_template.md

```
@@ -0,0 +1,12 @@
+# Pull Request Guidelines
+
+⚠️ **Warning**: This pull request will be **rejected** if the title does not follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary).
+
+### Examples of valid titles:
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)